bin/pnpm-install

```diff
@@ -52,7 +55,18 @@ function run (argv) {
     }
   })
 
-  return installCmd(cli.input, cli.flags).catch(require('../lib/err'))
+  var opts = Object.assign({}, getRC('npm'), getRC('pnpm'))
+
+  // This is needed because the arg values should be used only if they were passed
+  Object.keys(cli.flags).forEach(key => {
+    opts[key] = opts[key] || cli.flags[key]
+  })
+
+  return installCmd(cli.input, opts).catch(require('../lib/err'))
+}
+
+function getRC (appName) {
+  return camelcaseKeys(rc(appName))
```

Comment:
because the config keys are-named-like-this in npmrc. meow converts the flags to camel case, we convert the rc configs and then we merge the two sources of options together